.github/ISSUE_TEMPLATE/flaky_test.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Flaky test
+about: Report a flaky test failure
+title: 'Flaky-test: test_class.test_method'
+labels: ["component/test", "flaky-tests"]
```

Comment:
component/test doesn't exist. Please use `area/tests`. We can create the flaky-tests label @dlg99 @eolivelli

Reply:
Sure, thanks.

Reply:
@nicoloboschi  PTAL : )